modules/rosa-sts-creating-a-cluster-using-defaults-ocm.adoc

```diff
@@ -38,7 +38,7 @@ endif::[]
 +
 [NOTE]
 ====
-If your AWS account ID is not listed, check that you have successfully associated your AWS account with your Red Hat organization. If your account role ARNs are not listed, check that the required account-wide STS roles exist in your AWS account.
+If your AWS account ID is not listed, check that you have successfully associated your AWS account with your Red{nbsp}Hat organization. If your account role ARNs are not listed, check that the required account-wide STS roles exist in your AWS account.
 ====
 
 . Click *Next*.
```

Comment:
Can Can you fix the `====` on line 48? It should be at line 49 with the note at 48.

![image](https://github.com/openshift/openshift-docs/assets/16167833/440ddcef-9371-4472-94c0-1a6c30b334cd)